Assets/Scripts/PlayerController.cs

```diff
@@ -32,18 +41,40 @@ public class PlayerController : MonoBehaviour
 
     // Start is called before the first frame update
     void Start()
-    {
+    {   
         Canvas canvas = FindObjectOfType<Canvas>();
         controller = gameObject.AddComponent<CharacterController>();
         Debug.Log(this.transform.localPosition.x);
-        livesText.text = "Lives Remaining: " + health;
         baseColor = this.GetComponentInChildren<Renderer>().material.color;
         damageColor = Color.Lerp(baseColor, Color.red, 0.5f);
```

Comment:
Can make the color vars public and assign them in the inspector instead of in Start